open-telemetry/opentelemetry-dotnet-instrumentation
# Add update function to Powershell module

## Why

Fixes #3018 

## What

Adds `Update-OpenTelemetryCore` and `Get-OpenTelemetryInstallVersion` to Powershell Install module.

## Tests

* Download this script and install OpenTelemetry
* Download earlier version and replace `net\OpenTelemetry.AutoInstrumentation.dll`
* Run `Update-OpenTelemetryCore`
* Verify that last public version is installed

## Checklist



- [ ] New features are covered by tests.
